core/src/main/kotlin/org/sertain/hardware/Joysticks.kt

```diff
@@ -12,3 +12,31 @@ public fun Joystick.whenActive(buttonNumber: Int, command: Command) {
 public fun Joystick.whileActive(buttonNumber: Int, command: Command) {
     JoystickButton(this, buttonNumber).whileActive(command.mirror)
 }
+
+public fun Joystick.whenActive(buttonNumber: Int, block: () -> Unit) {
+    JoystickButton(this, buttonNumber).whenActive(
+        object : edu.wpi.first.wpilibj.command.Command() {
+            override fun execute() {
+                block()
+            }
```

Comment:
`block` returns Unit so would this work?

```kotlin
override fun execute() = block()
```

Reply:
`block()` should return a boolean